packages/tc-api-db-manager/__tests__/db-manager.spec.js

```diff
@@ -35,21 +49,25 @@ describe('creating db constraints', () => {
 	afterEach(() => jest.restoreAllMocks());
 
 	it("creates a uniqueness constraint if it doesn't exist", async () => {
-		mockConstraints(dbRun, [
-			'CONSTRAINT ON (s:Dog) ASSERT s.nose IS UNIQUE',
-		]);
+		mockConstraints({
+			stub: dbRun,
+			constraints: ['CONSTRAINT ON (s:Dog) ASSERT s.nose IS UNIQUE'],
+		});
 		schema.getTypes.mockReturnValue([
 			{ name: 'Dog', properties: { tail: { unique: true } } },
 		]);
 		await initConstraints();
 		expect(dbRun).toHaveBeenCalledWith(
 			'CREATE CONSTRAINT ON (s:Dog) ASSERT s.tail IS UNIQUE',
 		);
+		expect(dbRun).toHaveBeenCalledTimes(5);
 	});
-
 	// ignore this test until we upgrade to enterprise adiition - community edition doesn't have exists()
 	it.skip("creates an existence constraint if it doesn't exist", async () => {
```

Comment:
i'm not sure - I don't have the context on the upgrade - is this something that is going to happen?

Reply:
I think you could remove all the existence constraint stuff in a separate PR that's clearly named 'Remove existence constraint' so it's easy to find if we ever want to add it back in.

In the early days of the project it felt like an enterprise licence might be imminent, but it's turned out not to be the case - the api muddles on through without the benefits it would bring